modules/nf-core/bcftools/stats/tests/nextflow.config

```diff
@@ -0,0 +1,2 @@
+process {
+}
```

Comment:
Is this file needed?

Reply:
Probably not? I just ported from pytest so that might be why its there.